plugins/Over9000/__init__.py

```diff
@@ -0,0 +1,42 @@
+﻿from random import choice, randint
```

Comment:
Imports are incorrectly ordered. Imports should be split into the following 3 sections, each separated with a blank line and ordered alphabetically:

* Standard library imports
* site_packages imports (discord and discord.ext, in this case)
* App/package-specific imports (.quotes and Plugin, in this case)

Most editors provide functionality to re-order imports automatically.
* [VSCode](https://code.visualstudio.com/docs/python/editing#_sort-imports)
* [PyCharm](https://www.jetbrains.com/help/pycharm-edu/optimizing-imports.html)

Reply:
Fixed!